Monika After Story/game/script-topics.rpy

```diff
@@ -15749,3 +15749,54 @@ label monika_literature_value:
     m 1hksdlb "That's just my opinion as president of a literature club, though. {w=0.2}I guess most people wouldn't think that deeply about it."
     return
+    
+# Do you like nature?
+default persistent._mas_pm_likes_nature = None
+
+init 5 python:
+    addEvent(Event(persistent.event_database,eventlabel="monika_nature",category=['nature', 'you'],prompt="The outdoors",random=True))
+ 
+label monika_nature:
+    m 2esd "Hey, [player]..."
+    $ _history_list.pop()
+    menu:
+        m "Do you like nature{fast}?"
+        "Yes.":
+            $ persistent._mas_pm_likes_nature = True
+            m 2suo "Really? That's wonderful!"
+            m 3esd "You know, I think nature is something we should cherish."
+            m 3esa "Not only is it beautiful, but it also helps humanity out."
+            m 4esb "Insects pollinate our crops, trees give us wood and shade, pets offer us companionship..."
+            m 3esd "Most of all, autotrophs such as plants and algae produce food and oxygen; without them, most life on Earth wouldn't exist."
+            m 1esu "Because of that, I think it's only fair that we give something back to nature since it does so much for us."
+            m 7hub "So, here's Monika's Green Tip of the Day!"
+            m 7esc "Sometimes, people hesitate to go green because they're worried about that it's too expensive."
+            m 4eud "That's partially true. Electric vehicles, smart homes, and solar roofs can all cost a fortune."
+            m 3eub "However, you can make a difference and {i}save{/i} money just by making a few simple choices each day!"
+            m 3esb "Turning off appliances, taking shorter showers, buying a reusable water bottle, commuting by public transport..."
+            m 4esa "You could even buy a houseplant, or grow your own garden!"
+            m 7eud "Engaging in your local community can go a long way as well; if you take the initiative, others are sure to follow."
+            m 3esu "The important thing is to make a habit of thinking sustainably. If you can do that, you'll reduce your egological footprint in no time."
+            m 3wud "Who knows, maybe you'll even become happier and healthier on the way."
+            m 4esu "After all, a sustainable life is a satisfying life."
+            m 3eua "For me, at least."
+            m 3esb "That's my advice for today!"
+            m 2hka "Thanks for listening, [player]~"
+     
+        "No.":
+            $ persistent._mas_pm_likes_nature = False
+            m 1esa "That's okay, [player]."
+            m 3lsd "Not everyone enjoys the outdoors."
+            m 3euc "Some prefer the comfortable ambience of their homes, especially when technology makes them more convenient than ever."
+            m 1esd "Honestly, I can understand where they're coming from."
+            m 3eud "I spend most of my time reading, writing, coding, and being with you; all of that is easier to do indoors."
```

Comment:
When flirting a bit with the player, I think Monika should definitely smile.

Reply:
The poses are outdated because I've been focusing on the dialogue, but somehow I'm not tired yet so I can definitely give them a look over. I really appreciate the help, thank you—this topic would have been garbage if it had gone through on the first commit. <.>

Reply:
~~Oh no, I'm just the first guy, don't worry, you still got like 2 more guys to get through.~~

They're nice though, so don't worry.